apps/website/.vuepress/config.js

```diff
@@ -0,0 +1,85 @@
+const markdownItAttrs = require('markdown-it-attrs');
+const sidebar = require('./sidebar');
+const blocks = require('./blocks');
+const path = require('path');
+
+module.exports = {
+  title: 'Clarity Design System',
+  description: 'Clarity Design System is the premiere design system for application development.',
+  head: [['link', { rel: 'shortcut icon', href: '/favicon.ico' }]],
+  dest: '../../dist/website',
+  themeConfig: {
+    // algolia: {
+    //   apiKey: 'd3349af25a19e3824cb48e57fe88fdcb',
+    //   indexName: 'clarity',
+    // },
+    lastUpdated: false,
+    search: false, // Disabling for the moment
+    sidebar,
+  },
+  markdown: {
+    extendMarkdown: md => {
+      // Disabling include plugin until I figure out if it casuse Vuepress to hang.
+      // md.use(require('markdown-it-include'));
+      md.use(markdownItAttrs);
+    },
+    anchor: {
+      permalink: true,
+      permalinkBefore: false,
+      permalinkSymbol: '§',
+    },
+  },
+  extraWatchFiles: ['.vuepress/sidebar.js', '.vuepress/blocks.js'],
+  plugins: [
+    'vuepress-plugin-table-of-contents',
+    '@vuepress/active-header-links',
+    '@vuepress/last-updated',
+    'vuepress-plugin-smooth-scroll',
+    [
+      'vuepress-plugin-sitemap',
+      {
+        hostname: 'https://clarity.design',
+        outFile: 'sitemap.xml',
+      },
+    ],
+    [
+      '@vuepress/google-analytics',
+      {
+        ga: 'UA-86120402-1',
+      },
+    ],
```

Comment:
The current site uses clean url formats so we probably want to use something like this to preserve the URL structure and have better SEO https://vuepress.github.io/en/plugins/clean-urls/#installation. This plugin might simplify the `resolvePage` util function.

```
clarity.design/components/button/
clarity.design/components/button/demo.html
clarity.design/components/button/api.html

// clean urls
clarity.design/components/button
clarity.design/components/button/demo
clarity.design/components/button/api
```

```
[
  'vuepress-plugin-clean-urls',
  {
    normalSuffix: '',
    indexSuffix: '',
    notFoundPath: '/404',
  },
],
```

Reply:
Plugin added. Need to verify it after it deploys the preview. .